tokio-util/CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 0.4.0 (October 15, 2020)
+
+### Added
+- sync: `CancellationToken` for coordinating task cancellation (#2747).
+- rt: `TokioContext` sets the Tokio runtime for the duration of a future (#2791)
```

Comment:
Can you add a `0.3.2` entry, with this one? Since, I am going to release that version so we can support compat, would make sense to also add that onto master.

Reply:
This should be done as part of the 0.3.2 release.

Reply:
There are too many breaking changes to release a 0.3.2 so that will have to be as 0.4 but I have elected to implement what I wanted as an external crate. So this can be ignored for now.